app/src/main/res/layout/activity_splash.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think you mentioned earlier that this was changed away from a ConstraintLayout, but it seems like it's still one--can we make this a FrameLayout, instead?

Reply:
yes, its resolved